src/frontend/mame/clifront.cpp

```diff
@@ -507,14 +507,14 @@ void cli_frontend::listbrothers(const char *gamename)
 	}
 
 	// print the header
-	osd_printf_info("Source file:     Name:            Parent:\n");
+	osd_printf_info("Source file:         Name:            Parent:\n");
```

Comment:
Please change this to use the form:

```
   osd_printf_info("%-20s %-16s %s\n", "Source file:", "Name:", "Parent:\n");
```

This will allow people to just update the numbers next time, rather than counting spaces.